app/assets/javascripts/signup/email-value.coffee

```diff
@@ -27,6 +27,7 @@ class OX.Signup.EmailValue
 
 
   setType: (newUserType) ->
+    newUserType = if newUserType == "student" then "student" else "instructor"
     @group.find("[data-audience=\"#{@userType}\"]").hide()
```

Comment:
Could also be expressed as: `newUserType = 'instructor' unless newUserType is 'student'`